server/icat/include/icatHighLevelRoutines.hpp

```diff
@@ -137,7 +138,7 @@ int chlSetQuota( rsComm_t *rsComm, const char *type, const char *name, const cha
                  const char *limit );
 int chlCheckQuota( rsComm_t *rsComm, const char *userName, const char *rescName,
                    rodsLong_t *userQuota, int *quotaStatus );
-int chlDelUnusedAVUs( rsComm_t *rsComm );
+int chlDelUnusedAVUs( rsComm_t *rsComm ); // TODO Does this need a condInput too?
```

Comment:
investigate

Reply:
this function does not need a `condInput`.

it is invoked by `iadmin rum` which maps directly to `rxGeneralAdmin()` -> `chlDelUnusedAVUs()`. this is okay because the endpoint for `rxGeneralAdmin()` is protected by *apiTable.hpp*.

it's also directly invoked by `msiDeleteUnusedAVUs()`. this can be invoked by non-admins via the NREP, bypassing the check provided by *apiTable.hpp*.